packages/server-core/src/media/static-resource/static-resource.hooks.ts

```diff
@@ -165,25 +183,24 @@ export default {
         iffElse(
           checkScope('static_resource', 'read'),
```

Comment:
we need to add the admin action back such that we run the else condition here for all users, not just users who aren't admins. this will cause any find operations to give back different data for admins (specificlaly those with static_resource:read scopes)

Reply:
So we don't want admins to be able to view everything in assets? the action is built into the query. If we ever want an admin to actually see the assets within the studio we will have to query for admin and add into the action within the editor. I was just under the assumption we want admins to see everything within the studiol

Reply:
The action is for the admin panel. The studio should act the same for everyone.

Reply:
Sounds good to me